lib/permit_phoenix/live_view/authorize_hook.ex

```diff
@@ -227,7 +229,16 @@ defmodule Permit.Phoenix.LiveView.AuthorizeHook do
     |> Phoenix.LiveView.attach_hook(:params_authorization, :handle_params, fn params,
                                                                               _uri,
                                                                               socket ->
-      authenticate_and_authorize!(socket, session, params)
+      authenticate_and_authorize!(socket, socket.assigns.live_action, session, params)
+    end)
+    |> Phoenix.LiveView.attach_hook(:event_authorization, :handle_event, fn event,
```

Comment:
1. Name this function `attach_hooks`
2. Move `action_mapping` to the `LiveView` behaviour and name it `event_mapping`. Then load it here as `socket.view.event_mapping(event)`